app/services/manifest_validator.rb

```diff
@@ -0,0 +1,28 @@
+class ManifestValidator
+  include JsonSchemaHelper
+
+  attr_reader :content, :errors
```

Comment:
I'd prefer to test public facing methods only. the smaller the API of our objects, the better! if we think there is logic happening in private methods that is complicated enough to warrant its own test, perhaps its time to extract that into another object